azkaban-common/src/main/java/azkaban/utils/Utils.java

```diff
@@ -325,6 +327,18 @@ public static String formatDuration(long startTime, long endTime) {
     return days + "d " + hours + "h " + minutes + "m";
   }
 
+  public static void setCurrentTimeMillisFixed(long millis) {
+    currentTimeMillisFixed = millis;
+  }
+
+  public static void setCurrentTimeMillisSystem() {
```

Comment:
How about a name like useSystemTime? Or add a comment explaining what this is for?